pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -263,6 +263,12 @@ const (
 	// ShootOperationRotateSSHKeypair is a constant for an annotation on a Shoot indicating that the SSH keypair for the shoot
 	// nodes shall be rotated.
 	ShootOperationRotateSSHKeypair = "rotate-ssh-keypair"
+	// ShootOperationRotateCAStart is a constant for an annotation on a Shoot indicating that the rotation of the
+	// certificate authorities shall be started.
+	ShootOperationRotateCAStart = "rotate-ca-start"
+	// ShootOperationRotateCAComplete is a constant for an annotation on a Shoot indicating that the rotation of the
+	// certificate authorities shall be completed.
+	ShootOperationRotateCAComplete = "rotate-ca-complete"
```

Comment:
I would like to keep the trigger annotation naming with "start" and "complete" since this is easy to understand and more concise. Would this be okay? I see that others do it similarly ([ref](https://cloud.google.com/kubernetes-engine/docs/how-to/credential-rotation#initiating_the_rotation) and [ref](https://cloud.google.com/kubernetes-engine/docs/how-to/credential-rotation#completing_the_rotation)).

Reply:
ok with me